src/request/components/request-detail-panel-execution.jsx

```diff
@@ -222,61 +305,94 @@ var CommandList = React.createClass({
         var beforeExecuteCommandMessages = this.props.beforeExecuteCommandMessages;
         var afterExecuteCommandMessages = this.props.afterExecuteCommandMessages;
         var mongoDBMessages = this.props.mongoDBMessages;
+        var dataHttpRequestMessages = this.props.dataHttpRequestMessages;
```

Comment:
These 5 lines could be combined in a destructuring call to make it a little cleaner, e.g.

```
const {
  beforeExecuteCommandMessages,
  afterExecuteCommandMessages,
  ...
} = this.props;